src/openrct2/peep/peep.c

```diff
@@ -1392,12 +1392,14 @@ static uint8 peep_get_action_sprite_type(rct_peep* peep)
 	}
 }
 
-void sub_693B58(rct_peep* peep){
+void peep_update_current_action_sprite_type(rct_peep* peep){
```

Comment:
I find it easier to know that a given function was disassembled and whether any bugs found in it could have been original. Without such comment I can still dig this information up, but it takes much longer.